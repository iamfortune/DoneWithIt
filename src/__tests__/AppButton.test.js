import React from 'react';
import renderer from 'react-test-renderer';
import AppButton from '../../app/components/AppButton';

describe('Testing AppButton', () => {
    test('shows AppButton renders correctly', () => {
        const button = renderer.create( < AppButton / > ).toJSON();
        expect(button).toMatchSnapshot();
    });
});