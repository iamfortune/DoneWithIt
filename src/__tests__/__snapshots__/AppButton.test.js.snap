// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing AppButton shows AppButton renders correctly 1`] = `
<View
  accessible={true}
  focusable={false}
  isTVSelectable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fc5c65",
      "borderRadius": 25,
      "justifyContent": "center",
      "marginVertical": 10,
      "opacity": 1,
      "padding": 15,
      "width": "100%",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#fff",
        "fontSize": 18,
        "fontWeight": "bold",
        "textTransform": "uppercase",
      }
    }
  />
</View>
`;
